### ISSUE TITLE
Are the -m, --mol options not needed?

### ISSUE THREAD

**Issue by kohei-noda-qcrg** [RQC-HU/sum_dirac_dfcoef#12]
## Descriptions of this issue
- In the DIRAC output, the following data is written.
(e.g. UO2 2+)
```
  Atoms and basis sets
  --------------------

  Number of atom types :    2
  Total number of atoms:    3

  label    atoms   charge   prim    cont     basis
  ----------------------------------------------------------------------
  U           1      92     472     472      L  - [33s29p20d13f4g2h|33s29p20d13f4g2h]
  O           2       8      66      66      L  - [14s8p3d1f|14s8p3d1f]
  ----------------------------------------------------------------------
                            604     604      L  - large components
  ----------------------------------------------------------------------
  total:      3     108     604     604
```

If this program reads and parses the above data, does it not require the user to set the -m and -mol options?

## Concern

- If a user sets [ghost centers or/and point charges](https://www.diracprogram.org/doc/release-22/molecule_and_basis/molecule_using_xyz.html#including-ghost-centers-and-point-charges), I don't know what outputs will be and how they should be parsed.

**Comment by kohei-noda-qcrg** [comment]
Is it merely getting the labels and checking the labels are included in all the elemental symbols?

**Comment by kohei-noda-qcrg** [comment]
> Is it merely getting the labels and checking the labels are included in all the elemental symbols?
https://github.com/kohei-noda-qcrg/summarize_dirac_dfcoef_coefficients/issues/12#issuecomment-1508996739

No, the user sets the label, and can be any name.
https://diracprogram.org/doc/release-23/molecule_and_basis/molecule_using_mol.html#:~:text=The%20code%20will%20identify%20basis%20sets%20based%20on%20the%20nuclear%20charge%2C%20not%20the%20label

**Comment by kohei-noda-qcrg** [comment]
- This issue was resolved in #34
